cz.yaml

```diff
@@ -53,6 +53,11 @@ COUPON_NOT_VALID: Kupón {coupon} není platný nebo byl již použit
 COUPON_NWC_ADDED: Množství {coins} NWC bylo přidáno na váš účet.
 COUPON_PLACEHOLDER: Vložte kód kupónu
 COUPON_REDEEM: Vyzvednout kupón
+COUPON_TUTORIAL_STEP1: null
+COUPON_TUTORIAL_STEP2: null
+COUPON_TUTORIAL_STEP3: null
```

Comment:
COUPON_TUTORIAL_STEP3: Pokud byl kupón na NWC, budou okamžitě přičteny na tvůj účet.